From e765f4fb3af08739a325f3d28852caed8f7bb31e Mon Sep 17 00:00:00 2001
From: Duy Dang <duy.dang.yb@renesas.com>
Date: Wed, 21 Oct 2020 10:48:34 +0700
Subject: [PATCH 294/445] dt-bindings: display: rcar-du: Document R8A774[2|70]
 DU

Add device tree bindings for R8A7742 (RZ/G1H) and R8A77470 (RZ/G1C) DUs.

Signed-off-by: Duy Dang <duy.dang.yb@renesas.com>
---
 Documentation/devicetree/bindings/display/renesas,du.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Documentation/devicetree/bindings/display/renesas,du.txt b/Documentation/devicetree/bindings/display/renesas,du.txt
index 96cc080..e4d0a93 100644
--- a/Documentation/devicetree/bindings/display/renesas,du.txt
+++ b/Documentation/devicetree/bindings/display/renesas,du.txt
@@ -3,9 +3,11 @@
 Required Properties:
 
   - compatible: must be one of the following.
+    - "renesas,du-r8a7742" for R8A7742 (RZ/G1H) compatible DU
     - "renesas,du-r8a7743" for R8A7743 (RZ/G1M) compatible DU
     - "renesas,du-r8a7744" for R8A7744 (RZ/G1N) compatible DU
     - "renesas,du-r8a7745" for R8A7745 (RZ/G1E) compatible DU
+    - "renesas,du-r8a77470" for R8A77470 (RZ/G1C) compatible DU
     - "renesas,du-r8a7779" for R8A7779 (R-Car H1) compatible DU
     - "renesas,du-r8a7790" for R8A7790 (R-Car H2) compatible DU
     - "renesas,du-r8a7791" for R8A7791 (R-Car M2-W) compatible DU
@@ -46,9 +48,11 @@ corresponding to each DU output.
 
                       Port 0         Port1          Port2
 -----------------------------------------------------------------------------
+ R8A7742 (RZ/G1H)     DPAD 0         LVDS 0         LVDS 1
  R8A7743 (RZ/G1M)     DPAD 0         LVDS 0         -
  R8A7744 (RZ/G1N)     DPAD 0         LVDS 0         -
  R8A7745 (RZ/G1E)     DPAD 0         DPAD 1         -
+ R8A77470 (RZ/G1C)    DPAD 0         DPAD 1         LVDS 0
  R8A7779 (R-Car H1)   DPAD 0         DPAD 1         -
  R8A7790 (R-Car H2)   DPAD 0         LVDS 0         LVDS 1
  R8A7791 (R-Car M2-W) DPAD 0         LVDS 0         -
-- 
2.7.4

